nwb_conversion_tools/utils/nwbephyswriterdatachunkiterator.py

```diff
@@ -18,12 +30,50 @@ def __init__(
         chunk_mb: float = None,
         chunk_shape: tuple = None,
     ):
+        """
+        Initialize an Iterable object which returns DataChunks with data and their selections on each iteration.
+
+        Parameters
+        ----------
+        ephys_writer : si.BaseRecording
+          The BaseRecording object (from spikeinterface/spikeinterface) which handles the API for data transfer.
+        segment_index : int, optional
+          DESCRIPTION. The default is 0.
+        unsigned_coercion : list, optional
+          For unsigned data types (e.g., uint16), this value shifts the dtype to a signed type.
+          The default is None.
+        write_scaled : bool, optional
+          Whether to scale the data by the internal conversion factor ('gain', in spikeinterface).
+          The default is True.
```

Comment:
@Saksham20 Why is the default here `True`? For writing to NWB, `False` should _always_ be the default for applying the conversion factor.